plugins/rest/plugin.spec.yaml

```diff
@@ -223,7 +223,7 @@ actions:
       body:
         title: Body
         description: Payload to submit to the server when making the HTTP Request call
-        type: object
+        type: '[]object'
```

Comment:
Looks fine, but what's the behaviour of that input body if one would like to put only a single object? Does one have to put that inside `[]` brackets, i.e. `[{"key1": "value"}]` or it's possible to put a single JSON as well (i.e. `{"key1": "value"}`)? 😄

Reply:
Won't that affect all the workflows created before the update? I mean, if they update it, they all will have to change theirs input.